### DISCUSSION TITLE
Using helmfile with service account instead of kube-context

### DISCUSSION THREAD

**Discussion by yuvalsimon** [helmfile/helmfile#907] [Questions]
I want to use helmfile inside a pod that has kubernetes api access through service account. I didn't find how to get it to work without creating dummy kube-context. Is there a way to do it in the current release of helmfile?

**Comment by yxxhero** [comment]
@yuvalsimon I will work on this issue.

**Reply by yuvalsimon** [reply]
Thanks!
